src/plugins/dashboard/public/dashboard_actions/filters_notification_popover.tsx

```diff
@@ -126,7 +128,7 @@ export function FiltersNotificationPopover({ api }: { api: FiltersNotificationAc
         )}
       </EuiForm>
       <EuiPopoverFooter>
-        {!disableEditbutton && (
+        {!disableEditbutton && parentViewMode === 'edit' && (
```

Comment:
Can we wrap the entire footer with the `parentViewMode === 'edit'` so that we don't have the weird breaking line? Or just disable the button in view mode 🤔